test/runtests.jl

```diff
@@ -121,19 +122,62 @@ end
     @test length(testlogger.logs) == 2
 end
 
+@testset "DatetimeRotatingFileLogger" begin
+    mktempdir() do dir
+        drfl_sec = DatetimeRotatingFileLogger(dir, raw"\s\e\c-YYYY-mm-dd-HH-MM-SS.\l\o\g")
+        drfl_min = DatetimeRotatingFileLogger(dir, raw"\m\i\n-YYYY-mm-dd-HH-MM.\l\o\g")
+        sink = TeeLogger(drfl_sec, drfl_min)
+        with_logger(sink) do
+            while millisecond(now()) < 100 || millisecond(now()) > 200
+                sleep(0.001)
+            end
+            @info "first"
+            @info "second"
+            sleep(0.9)
+            @info("third")
+        end
+
+        # Drop anything that's not a .log file or empty
+        files = sort(map(f -> joinpath(dir, f), readdir(dir)))
+        files = filter(f -> endswith(f, ".log") && filesize(f) > 0, files)
+        sec_files = filter(f -> startswith(basename(f), "sec-"), files)
+        @test length(sec_files) == 2
+
+        min_files = filter(f -> startswith(basename(f), "min-"), files)
```

Comment:
Not a reflection on this code, but a complete aside.
If we were in julia-1.5 this would be so much nicer.

```suggestion
        files = sort([f in readdir(dir, join=true) if  endswith(f, ".log") && filesize(f) > 0])
        sec_files = filter(startswith("sec-") ∘ basename, files)
        @test length(sec_files) == 2

        min_files = filter(startswith("min-") ∘ basename, files)
```